sort.php

```diff
@@ -13,19 +13,34 @@ function checkForNextLine($arr)
     return $arr;
 }
 
-function bubbleSort($sort_array)
+function bubbleSort($sort_array, $options = false)
 {
     $array_indexes = count($sort_array);
     $sorted = false;
 
     for ($i = 0; $i < $array_indexes; $i++) {
         for ($j = 0; $j < $array_indexes - $i - 1; $j++) {
-            if ($sort_array[$j + 1] < $sort_array[$j]) {
-                $tmp = $sort_array[$j + 1];
-                $sort_array[$j + 1] = $sort_array[$j];
-                $sort_array[$j] = $tmp;
-
-                $sorted = true;
+            if ($options) {
+                if (strlen($sort_array[$i]) === 1) {
+                    $tmp = $sort_array[$i];
+                    $sort_array[$i] = $sort_array[0];
+                    $sort_array[0] = $tmp;
+                }
```

Comment:
What's this for?

Reply:
Since all elements in the array are compared as int the empty rows are also converted to an int and an empty row has the value 1 in integer and 1 greater than 0 and then the empty rows are sorted at the second position instead of at the beginning.

Reply:
I could also ignore the empty rows but in the standard sort func are also not ignored.

Reply:
An empty _line_ is not a valid int. Please don't care for such.

Reply:
Should I delete the code block?

Reply:
Exactly.